conceptnet5/builders/combine_assertions.py

```diff
@@ -152,15 +144,16 @@ def handle_file(self, input_filename, output_file):
         combine_assertions([input_filename], output_file, self.license)
 
 
+@click.command()
+#tab-separated csv file to be grouped into assertion
+@click.argument('input', type=click.Path(readable=True, dir_okay=False))
+#msgpack stream of assertions
+@click.argument('output', type=click.Path(writable=True, dir_okay=False))
+def cli(input, output):
+    combine_assertions(input,output)
```

Comment:
I think you could just decorate the `combine_assertions` function, because the `cli` function is identical to it.